TransformationSystem/ConfigTemplate.cfg

```diff
@@ -180,5 +180,24 @@ Agents
     BulkSubmission = false
   }
   ##END
+  ##BEGIN DataRecoveryAgent
+  DataRecoveryAgent
+  {
+    PollingTime = 3600
+    EnableFlag = False
+    MailTo =
+    MailFrom =
+    # List of TransformationIDs that will not be treated
+    TransformationsToIgnore =
+    # List of Transformation Statuses to treat
+    TransformationStatus = Active, Completing
+    # List of transformations that do not have input data
+    TransformationsNoInput = MCGeneration
+    # List of transformations that do have input data
+    TransformationsWithInput = MCReconstruction, MCSimulation, MCReconstruction_Overlay
```

Comment:
Other TS agents use these locations:
```
    transformationTypes = Operations().getValue('Transformations/DataProcessing', [])
    extendableTTypes = Operations().getValue('Transformations/ExtendableTransfTypes', ['MCSimulation'])
```
Which might not cover fully your case here, but nevertheless if more options need to be added should be in `/Operations/Transformations`.

Reply:
It is safe to assume that `ExtendableTransfTypes` contains transformations without input data, right? Thus `DataProcessing - ExtendableTransfTypes` gives me transformationsWithInput

Reply:
I would not think it's a 100% safe assumption. It works in LHCb and IIUC in CLIC but not sure about other VOs.

Reply:
Though can I use those assumption for the default values?
Things can be overwritten in the agent configuration.